tcmur_device.c

```diff
@@ -20,6 +20,7 @@
 #include "tcmu-runner.h"
 #include "tcmur_device.h"
 #include "tcmur_cmd_handler.h"
+#include "tcmu_runner_priv.h"
```

Comment:
No this header file exist.

Reply:
Forgot to git add. Will fix on the re-push.

Reply:
Build succeeded.

Reply:
Yeah, it works now :-)